server/src/main/java/io/crate/analyze/CopyStatementSettings.java

```diff
@@ -90,4 +120,32 @@ public static <E extends Enum<E>> E settingAsEnum(Class<E> settingsEnum, String
         }
         return Enum.valueOf(settingsEnum, settingValue.toUpperCase(Locale.ENGLISH));
     }
+
+    public static List<String> commonCopyToSettings = List.of(
+        COMPRESSION_SETTING.getKey(),
+        OUTPUT_FORMAT_SETTING.getKey(),
+        WAIT_FOR_COMPLETION_SETTING.getKey()
+    );
+
+    public static List<String> commonCopyFromSettings = List.of(
```

Comment:
Static vars should be all upper case.

Reply:
I will change this in the current open PR